manta-pay/concrete-protocol.tex

```diff
@@ -4,46 +4,79 @@
 
 \subsection{Poseidon Permutation}
 
-The \Poseidon{} Permutation \cite{KRRS21USENIX} is a finite field cryptographic primitive that can be used in lots of different contexts, like hash functions, commitment schemes, and symmetric encryption. \Poseidon{} plays a fundamental role in simplifying the protocol and reducing the overall cost of the Zero-Knowledge circuits. \Poseidon{} (without sponges) is a family of hash functions with the following signature:
-\[\Poseidon_k : \mathbb{F}^k \to \mathbb{F}\]
-over some sufficiencly large finite field $\mathbb{F}$. We make use of \Poseidon{} for a few values of $k$ in the concrete protocol below.
+The \Poseidon{} Permutation \cite{KRRS21USENIX} is a finite field cryptographic primitive that can be used in lots of different contexts, like hash functions, commitment schemes, and symmetric encryption. \Poseidon{} plays a fundamental role in simplifying the \Transfer{} protocol and reducing the overall cost of the Zero-Knowledge circuits. \Poseidon{} (without sponges) is a family of hash functions with the following signature:
```

Comment:
`signature` in this sentence refers to a different concept from the `Signature Scheme`, which appears a lot in the spec. Can we change the `signature` here to other terms (e.g., interface)?